frontend/channel/backend-messaging-service.ts

```diff
@@ -22,6 +22,9 @@ export class BackendMessagingService {
     });
 
     this.backgroundPageConnection.onMessage.addListener((message: any) => {
+      if (message.from && message.from === 'content-script') {
```

Comment:
This actually happens every time any content loads, regardless of whether it's a refresh, or a new page load, or even if the application is one being inspected or not. Probably not ideal.